### PR TITLE
Add empty state to search.

### PR DESCRIPTION
**Pull request by zackhall**
Adds an empty state if the search term has no matches. Like this:

image

### CODE REVIEW

File: assets/js/docs.js [hunk 1]
```diff
@@ -96,7 +96,9 @@
           })
           .value();
 
-      var collections = _.map(filtered, function(collection) {
+      var isCollectionEmpty = _.every(filtered, { 'visible': false });
+
+      var collections = .map(filtered, function(collection) {
```
Comment (jdalton) [inline assets/js/docs.js]: typo

Comment (zackhall) [inline assets/js/docs.js]: Where?

Comment (jdalton) [inline assets/js/docs.js]: ``js
var collections = .map(
```

Comment (zackhall) [inline assets/js/docs.js]: Yeah, how did I let that slip in? :-P

Comment (zackhall) [inline assets/js/docs.js]: Oh, bc I needed to refresh a few times to break the cache.

**Comment by jdalton** [comment]
🎉

**Pull request merged**